toolkit/imageconfigs/full.json

```diff
@@ -7,8 +7,12 @@
                 "packagelists/virtualization-host-packages.json",
                 "packagelists/core-packages-image.json",
                 "packagelists/hyperv-packages.json",
-                "packagelists/ssh-server.json"
+                "packagelists/ssh-server.json",
```

Comment:
As we discussed, I'm not convinced we want to port these changes to the existing images. Not strongly opposed, but "do no harm" applies here.

Reply:
If you decide to leave the configs alone, you can just remove the changes completely. The kernel command line entry can just be ommited and it will default to `""` which disables the selinux mode.

Reply:
hmmm, I've thought about this quite a bit and I think it may be best to leave selinux in here. I want to  test the policy in permissive mode across multiple configurations before backporting chris pebenitos enforcing changes. With the eventual goal of having it enforcing I think this is a necessary step? I totally agree in the "do no harm" approach and I think this first step of permissive mode is very close to it. Let me know if you disagree as I could be wrong here.